navit/maptool/CMakeLists.txt

```diff
@@ -1,16 +1,41 @@
-
 if(BUILD_MAPTOOL)
+
+	find_package(Protobuf-c REQUIRED)
+
+  # cmake_policy(SET CMP0026 OLD)
```

Comment:
Old comments that are not needed anymore

Reply:
in fact they will need later.

Reply:
This is a policy we should not need to set, we should only fix the problem, but okay
for now

Reply:
yes i've to remove the function and call the implicit mecanism but for now i don't have understand that well, see that later.

Reply:
#666 fixes this problem

Reply:
line 5, yes